blockchain_nodes/ensure_bitcoind.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+case "$OSTYPE" in
+  darwin*)  OS="OSX"; url="https://bitcoincore.org/bin/bitcoin-core-0.17.0/bitcoin-0.17.0-osx64.tar.gz";;
+  linux*)   OS="Linux"; url="https://bitcoincore.org/bin/bitcoin-core-0.17.0/bitcoin-0.17.0-x86_64-linux-gnu.tar.gz";;
+  *)        echo "unknown: $OSTYPE. Sorry, this is currently not supported"; exit 1 ;;
+esac
+
+
+TARGET_FOLDER=./blockchain_nodes/bitcoin
+
+if [ ! -d "${TARGET_FOLDER}" ]
+then
+    mkdir -p "${TARGET_FOLDER}"
+fi
+
+TARGET_FILE="${TARGET_FOLDER}/bitcoin-0.17.0"
+if [ -d "$TARGET_FILE" ]; then
+  echo "${TARGET_FILE} exists; all good mate!"
```

Comment:
I'd be in favor of not adding spam like this that we are gonna see each time we run the tests :)

Reply:
I always smile when reading this message. It's like a good friend who pats you on the back and says: `All good mate!`